spec/rspec/matchers/built_in/operators_spec.rb

```diff
@@ -228,22 +228,48 @@ def method_missing(name, *args, &block)
   end
 
   describe RSpec::Matchers::BuiltIn::PositiveOperatorMatcher do
+    let(:o) { Object.new }
+    before { def o.send(*_args); raise "DOH! Library developers shouldn't use #send!"; end }
+
     it "works when the target has implemented #send" do
-      o = Object.new
-      def o.send(*_args); raise "DOH! Library developers shouldn't use #send!" end
       expect {
         o.should == o
       }.not_to raise_error
     end
+
+    it "complains when negated", :if => RUBY_VERSION.to_f >= 1.9 do
```

Comment:
We'll be [removing](https://github.com/rspec/rspec-core/issues/1821) :if/:else in the next major.
```suggestion
    it "complains when negated", :skip => RUBY_VERSION.to_f < 1.9 do
```